libpfc-timer.cpp

```diff
@@ -69,7 +70,7 @@ static void inner_init(Context &context) {
         throw std::runtime_error(std::string("pfcInit() failed (error ") + std::to_string(err) + ": " + msg + ")");
     }
 
-    err = pfcPinThread(0);
+    err = pfcPinThread(pinned_cpu.Get());
     if (err) {
         // let's treat this as non-fatal, it could occur if, for example
         context.err() << "WARNING: Pinning to CPU 0 failed, continuing without pinning" << endl;
```

Comment:
The `if (err)` message and the `else` messages are wrong now since they refer to hardcoded 0.